packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -137,7 +137,6 @@ library TaikoData {
         uint64 timestamp; // slot 6 (90 bits)
         uint16 tier;
         uint8 contestations;
-        bytes32[4] __reserved;
```

Comment:
`TransitionState` and `Block` are only stored in mappings and are never used as an input, so don't really necessarily need these reserved gaps. Unless if it's to maintain compatibility with onchain smart contracts using this data, then they should be kept because then still useful I think.